clients/src/main/java/org/apache/kafka/clients/consumer/internals/events/ApplicationEventHandler.java

```diff
@@ -99,18 +100,17 @@ public long maximumTimeToWait() {
      *
      * <p/>
      *
-     * See {@link CompletableApplicationEvent#get(Timer)} and {@link Future#get(long, TimeUnit)} for more details.
+     * See {@link ConsumerUtils#getResult(Future, Timer)} and {@link Future#get(long, TimeUnit)} for more details.
      *
      * @param event A {@link CompletableApplicationEvent} created by the polling thread
-     * @param timer Timer for which to wait for the event to complete
      * @return      Value that is the result of the event
      * @param <T>   Type of return value of the event
      */
     public <T> T addAndGet(final CompletableApplicationEvent<T> event, final Timer timer) {
         Objects.requireNonNull(event, "CompletableApplicationEvent provided to addAndGet must be non-null");
         Objects.requireNonNull(timer, "Timer provided to addAndGet must be non-null");
         add(event);
-        return event.get(timer);
+        return ConsumerUtils.getResult(event.future(), timer);
     }
```

Comment:
The `CompletableApplicationEvent.get()` method was removed. Since it was just a shim over `ConsumerUtils.getResult()` anyway, I removed it to avoid confusion/misuse.